docs/tequila-api.md

```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is to be registered and have your API key.
+Check the Guide: [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
+
+To implement the basic search, you need the `/locations` and `/search` API calls.
+
+- **Locations API**: Search for cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
+
+- **Search API**: This API allows you to search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
+
+- **Multicity API**: This API provides the multicity search. The multicity trip is a journey where the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
+
+- **NOMAD API**: This search API is for those travelers, who would like to visit n-number of cities in any order. ([docs](https://tequila.kiwi.com/portal/resources/nomad_api))
+
+In order to finish booking the itinerary, you need to call the `/booking` API.
+
+- **Booking API**: This API allows you to check if the price of the itinerary is still valid and if the itinerary is still available. It also saves the booking which initiates the booking process and confirms that the user payment was successfully. ([docs](https://tequila.kiwi.com/portal/resources/booking_api))
+
+- **Webhooks API**: This API is a way of how we provide other applications with real-time information. ([docs](https://tequila.kiwi.com/portal/resources/webhooks))
```

Comment:
```suggestion
- **Webhooks API**: This API is a way for <span>Kiwi</span>.com to provide other applications with real-time information. ([docs](https://tequila.kiwi.com/portal/resources/webhooks))
```
I find the documentation on Tequila very un-informative and it's difficult to understand what it's about.